### PR TITLE
Multipage View, Login and npm commands

### PR DESCRIPTION
**Pull request by gleal42**
## Router, Login Page and npm commands 
1. Created router with 4 views
2. Changed main.js to main.ts
3. Incorporated Chat into one of the Views

## Related issues:
- #7

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -10,8 +10,8 @@
     <div v-if="islogged">
       <RouterView/>
     </div>
-    <div v-else @isbuttonclicked="login">
-      <Login/>
+    <div v-else >
```
Comment (gleal42) [inline frontend/src/App.vue:13]: Whoops, this was causing the error in login

Comment (ncameiri) [reply frontend/src/App.vue:13]: `    <div v-else @click="login()">`

**Review by ncameiri — APPROVED** [review]
## Issue(s) found:
- Login works good and chat has been incorporated inside Chat route


## Style related issue(s) found:
N/A

## Notes:
N/A

File: frontend/src/components/LoginPage.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <div class="welcome">
+        <h1>Welcome to Raquetas</h1>
+        <button class="button" @click="buttonClick" >Login</button>
+    </div>
+  </div>
+</template>
+
+
+<script setup lang="ts">
+const emit = defineEmits(['isbuttonclicked'])
```
Comment (Dmarceli) [inline frontend/src/components/LoginPage.vue:12]: why do we need emits?? from what i understand they are mostly used for Parent-Child Communication

**Review by Dmarceli — APPROVED** [review]
LGTM

**Pull request merged**